papermill/tests/test_api.py

```diff
@@ -36,6 +36,10 @@ def test(self):
         )
         assert_frame_equal(nb.dataframe, expected_df)
 
+        with self.assertRaises(PapermillException) as excinfo:
+            nb.display_output('foo')
+        assert "Output Name 'foo' is not available in this notebook." in str(excinfo.exception)
```

Comment:
Does excinfo.value not work per this [`pytest.raises`](https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions) example?